### PR TITLE
feat: create component PartyInfo

### LINKED ISSUE #38: Adicionar componente PartyInfo

### PR DESCRIPTION
**Pull request by Joel-leal**
# Description

This component will display the selected party data, rendering on screen the following information:

* candidate: This is the main candida

* vice: This is the alternate candidate

* party: Will be filled with the political party

Screen Shot 2022-06-30 at 14 07 32

## Changes

- Create the PartyInfo component
- Show on the voting screen

## Board issue

(include here the board issue number to which this PR refers to)

- Closes #38

### CODE REVIEW

File: packages/components/PartyInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Box, Text } from '@chakra-ui/react';
+
+interface IPartyInfo {
```
Comment (frattezi) [inline packages/components/PartyInfo.tsx:3]: Minor

I beliave you could move this interface to our `/entities` folder in a new file, problably we could call the file election.
So, create a new file `/entities/election` and move this inteface there, so we keep our interfaces organized!

File: packages/components/PartyInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <Box
+      display="flex"
+      flexDirection="column"
+      border="2px solid black"
+      w="35vw"
+      h="35vh"
+      justifyContent="center"
+      alignItems="left"
+    >
+      <Box display="flex" marginLeft="10%" py="20px">
```
Comment (frattezi) [inline packages/components/PartyInfo.tsx:20]: Why is a marginLeft here? We should avoid margins, shouldn't this be a Box with 100% width?

**Review by frattezi — COMMENTED** [review]
Just some minor issues, but it was a great evolution from the last task!
Keep it like that 🚀

**Comment by frattezi** [comment]
@Joel-leal let's focus also o finishing this PR since the comments are really simple and it's almost done!

File: packages/components/PartyInfo.tsx (hunk 1 above)
Comment (frattezi) [reply packages/components/PartyInfo.tsx:3]: Acredito que neste caso, como a interface não será re utilizada podemos mante-la aqui.

**Review by frattezi — DISMISSED** [review]

**Review by frattezi — APPROVED** [review]

File: packages/components/PartyInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Box, Text } from '@chakra-ui/react';
+
+interface IPartyInfo {
+  candidate?: string;
+  vice?: string;
+  partido?: string;
```
Comment (borgesgfj) [inline packages/components/PartyInfo.tsx:6]: `party?: string;`

File: packages/components/PartyInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+export default function PartyInfo({ candidate, vice, partido }: IPartyInfo) {
```
Comment (borgesgfj) [inline packages/components/PartyInfo.tsx:9]: A typo error here, change `partido` by `party`

**Pull request merged**